<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>예외 처리 구문 내부에서 return 사용하기(1)</title>
    <script>
        function test() {
            try {
                alert('A 위치')
                throw "에외 강제 발생" // throw 키워드 : 예외를 강제로 발생
            } catch (e) {
                alert('B 위치')
                return
            }
            alert('C 위치')
        }
        // 함수 호출
        test()



    </script>
</head>
<body>

</body>
</html>